### DISCUSSION TITLE
How can I implement Credentials Provider in app router

### DISCUSSION THREAD

**Discussion by melodyxpot** [nextauthjs/next-auth#8763] [Help]
### Summary

Hello everyone.
I am making some next.js project and I use app router.
I tried to use next-auth to implement authentication but I couldn't find the exact guide or sample, example for next.js app router + credentials provider.
Please guide me.
How can I implement it on app router?
Thank-you.

**Comment by MihirModi1421** [comment]
@melodyxpot have you found any reference or solution to implement it.

**Reply by MihirModi1421** [reply]
also i am having seperate nest js backend.

**Comment by Bee1130** [comment]
have you found any reference or solution to implement it.
